benchmark-scripts/benchmark.sh

```diff
@@ -335,7 +335,7 @@ do
               fi
             fi
           else
-            # since there is no longer --rm automatically remove docker-run containers
+            # since there is no longer --rm automatically remove exited run containers
```

Comment:
do we need to mention about containers here since we are no longer using docker run? maybe we can remove this comment totally

Reply:
good catch, for ovms-client docker being removed, we don't need actually do removing of docker container of that now

Reply:
sorry, actually, this is in `else` branch so it is for dlstreamer case and we still have docker containers running from run-dlstream.sh; so the comment is still valid